src/components/donate/HeroSection.tsx

```diff
@@ -0,0 +1,94 @@
+import { Flex, Heading, Text, Button, Box } from "@radix-ui/themes";
+import Image from "next/image";
+import { relative } from "path";
+const HeroSection = () => {
+  return (
+    <>
+      {/* blue bg */}
+      <Flex
+        mb={"20%"}
+        position={"relative"}
+        height={"550px"}
+        style={{ background: "#1D2F73" }}
+        className="shadow-xl"
+      >
+        <Flex
+          position={{ sm: "absolute" }}
+          top={{ sm: "0" }}
+          justify={"center"}
+          width={"100%"}
+          pl={{ sm: "25%" }}
+        >
+          <Flex>
+            <Box position={"relative"}>
+              <Image
+                width={"1000"}
+                height={"1000"}
+                src="/images/donate/european-refugee-relief-hero-tinted.png"
+                alt="european-refugee-relief-hero-tinted"
+              />
+
+              <Flex
+                direction={"column"}
+                position={"absolute"}
+                bottom={"0"}
+                left={{ sm: "-27%" }}
+                gap={"7"}
+                px={"4"}
+                width={"100%"}
+              >
+                <Box width={"600px"}>
+                  <Text size={{ initial: "8", sm: "9" }} className="text-white">
```

Comment:
**nitpick:** As the page title, this should probably be a `Heading` component with the `as` attribute set to `h1`.